lib/src/facts/resolver.cc

```diff
@@ -24,6 +26,23 @@ namespace facter { namespace facts {
                 throw invalid_name_pattern_exception(ex.what());
             }
         }
+
+#ifdef LEATHERMAN_USE_LOCALES
+        // build Accept-Language list for HTTP-based resolvers
```

Comment:
Sure, on-demand makes sense. I had been considering that earlier (or a derived class for HTTP-based resolvers) and then forgot about it. I'll implement the on-demand version.